server/models/Project.ts

```diff
@@ -3,7 +3,8 @@ import mongoose, { Schema } from 'mongoose';
 const projectSchema = new Schema({
     userID: { type: String, required: true },
     name: { type: String, required: true },
-    queries: { type: Array, required: false },
```

Comment:
Projects aren't storing queries?

Reply:
Queries are storing their associated project ID, and same with projects storing userIDs